tests/test.spec.js

```diff
@@ -0,0 +1,74 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import addReducer from '../index';
+import profileReducer from './reducers/profile.reducer';
+import cartReducer from './reducers/cart.reducer';
+
+function testAddNameReducer(store) {
+    store.dispatch({
+        type: 'ADD_NAME',
+        name: 'Rick Sanchez'
+    });
+
+    expect(store.getState().profile).to.deep.equal({
+        name: 'Rick Sanchez'
+    });
+}
+
+describe('Redux Store', () => {
+    let store;
+    beforeEach(() => {
+        store = createStore((state) => state);
+    });
+
+    it('should be an empty store', () => {
+        expect(store.getState()).to.equal(undefined);
+    });
+
+    describe('should add one reducer', () => {
+        beforeEach(() => {
+            addReducer(store, 'profile', profileReducer);
+        });
+
+        it('store should be initialized empty', () => {
```

Comment:
a better wording would be something along those lines:
`store should be initialized with initial state`